index.js

```diff
@@ -32,7 +40,7 @@ function render(st = state.Home) {
 function eventListenerBundler(st) {
   listenForLogin(st);
   listenForSignup(st);
-  storyComplete(st);
+  lessonComplete(st, "story");
```

Comment:
ideally you'd do something like:

`allthelessons.forEach(lesson => lessonComplete(st, lesson))`

Reply:
yes I was trying to do that as I think it would be much easier but I just went with the indivual story function because something was giving me issues